File: Encapsulation-Polymorph-HW/Bank/Customer.cs
using System;

namespace Bank
{
    class Customer
    {
        private string type;

        public Customer(string type)
        {
            this.Type = type;
        }

        public string Type
        {
            get
            {
                return this.type;
            }
            set
            {
                this.type = value;
            }
        }
    }
}

File: OOP-Java/src/Shop/Customer.java
package Shop;

public class Customer {
	private String name;
	private int age;
	private double balance;
	
	public Customer(String name, int age, double balance) {
		super();
		this.name = name;
		this.age = age;
		this.balance = balance;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public double getBalance() {
		return balance;
	}

	public void setBalance(double balance) {
		this.balance = balance;
	}
	
	@Override
	public String toString() {
		return "Customer [name=" + name + ", age=" + age + ", balance="
				+ balance + "]";
	}
	
	
	
	
}
